x-pack/solutions/security/plugins/security_solution/public/common/components/with_data_view/index.tsx

```diff
@@ -40,7 +40,7 @@ export const withDataView = <P extends WithDataViewArg>(
       sourcererScope: SourcererScopeName.timeline,
     });
 
-    const { newDataViewPickerEnabled } = useEnableExperimental();
+    const newDataViewPickerEnabled = useIsExperimentalFeatureEnabled('newDataViewPickerEnabled');
```

Comment:
what prompted the switch?

Reply:
I like the api more but its subjective, it works the same way. When mocking though, it lets you just return boolean.